core/src/main/scala/ox/retry/retry.scala

```diff
@@ -0,0 +1,81 @@
+package ox.retry
+
+import scala.concurrent.duration.*
+import scala.util.{Random, Try}
+
+sealed trait Jitter
+
+object Jitter:
+  case object None extends Jitter
+  case object Full extends Jitter
+  case object Equal extends Jitter
+  case object Decorrelated extends Jitter
```

Comment:
This could be a Scala3 enum

```suggestion
enum Jitter:
  case None, Full, Equal, Decorrelated

```

Reply:
Good catch, fixed, thanks 😄